sdk/core/System.ClientModel/tests/Pipeline/ClientLoggingPolicyTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Threading.Tasks;
+using ClientModel.Tests;
+using ClientModel.Tests.Mocks;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using SyncAsyncTestBase = ClientModel.Tests.SyncAsyncTestBase;
+
+namespace System.ClientModel.Tests.Pipeline;
+
+public class ClientLoggingPolicyTests : SyncAsyncTestBase
+{
+    private const int RequestEvent = 1;
```

Comment:
Do we need these constants?   I may be overlooking something, but I don't see them used.